modules/core/shared/src/main/scala/coursier/core/DependencySet.scala

```diff
@@ -103,11 +100,11 @@ object DependencySet {
 
 
   private object Sets {
-    def empty[T]: Sets[T] = Sets(TreeMap.empty, Map.empty, Map.empty)
+    def empty[T]: Sets[T] = Sets(IntMap.empty, Map.empty, Map.empty)
   }
 
   private final case class Sets[T] private (
-    required: TreeMap[Int, Set[T]],
+    required: IntMap[Set[T]],
```

Comment:
this change broke Mima, but this is a private class...

Reply:
This looks like a false positive, I just pushed a commit to ignore those.